playerRatings/playerRatings.py

```diff
@@ -390,35 +390,35 @@ async def get_ordered_opponent_names_and_seeds(self, ctx, seed, is_home, opposin
         sorted_opponents = await self.sort_members_by_rating(ctx, active_opponents)
         if is_home:
             if seed == 1:
-                ordered_opponent_names.append(sorted_opponents[2].nick)
-                ordered_opponent_names.append(sorted_opponents[1].nick)
-                ordered_opponent_names.append(sorted_opponents[0].nick)
+                ordered_opponent_names.append(sorted_opponents[2].display_name)
```

Comment:
I've had issues using `Member.display_name` in the past. It _should_ be supported based on the current running API documentation, but it would be best to test it first.

If it proves to fail, this would be my alternative:
```py
try:
    opponent_name_1 = sorted_opponents[0].nick
except:
    opponent_name_1 = sorted_opponents[0].name
```
This could quickly get lengthy and redundant, and could be extracted to a helper function.

That said, in _this_ scenario specifically, each opponent should have a nickname that aligns with their franchise or FA status.

Reply:
We're going to push the change for now and monitor if the issue happens again